re.h

```diff
@@ -53,10 +58,12 @@ re_t re_compile(const char* pattern);
 int  re_matchp(re_t pattern, const char* text, int* matchlenght);
 
 /* Free memory of the compiled pattern */
+#if defined(RE_ENABLE_MULTI_PATTERNS) && (RE_ENABLE_MULTI_PATTERNS == 1)
 void re_freecompile(re_t pattern);
+#endif
 
 /* Find matches of the txt pattern inside text (will compile automatically first). */
-int  re_match(const char* pattern, const char* text, int* matchlenght);
+int re_match(const char *pattern, const char *text, int *matchlenght);
```

Comment:
It's better to avoid mixing functional and style changes in one commit. This change also breaks style consistency with the rest of the code, doesn't it? Patches should preserve existing code style even if it's not your favorite one.

Reply:
you are right, fix it.